src/main/java/homework4/Main.java

```diff
@@ -0,0 +1,18 @@
+package homework4;
+
+import java.util.HashMap;
+
+public class Main {
+    public static void main(String[] args) {
+        Bank<Integer, Asset> bank = new Bank<>("Volodymirska 16");
```

Comment:
You second time repeat same. Ok. Don't you any collections in your homework. Look at HW other students. I would like how you can work with generics. You use there Map<k,v> logic